solidity/test/StandardPoolConverter.js

```diff
@@ -1188,4 +1196,212 @@ describe('StandardPoolConverter', () => {
             });
         }
     });
+
+    describe('verifies that the network fee is transferred correctly via', () => {
+        const ONE_TOKEN = new BN(10).pow(new BN(18));
+        const TOTAL_SUPPLY = ONE_TOKEN.muln(1000000);
+        const CONVERSION_AMOUNT = ONE_TOKEN.muln(100);
+
+        for (const initialBalance1 of [100000, 200000, 400000, 800000]) {
+            for (const initialBalance2 of [100000, 300000, 500000, 700000]) {
+                for (const conversionFeePercent of [0, 5, 10, 25, 75]) {
+                    for (const networkFeePercent of [0, 5, 10, 25, 75]) {
```

Comment:
Can you also check 100%, just to be on the safe side? It's as unrealistic as 0%, but can uncover potentially dangerous corner case (which can be maliciously triggered by the owner of the converter)

Reply:
Done.

Reply:
BTW, it increases the duration of this already-long test by 20% (checking 6 different network fees instead of 5).

Reply:
That's a small price to pay